### PR TITLE
Adding new `ProjectHook` attributes:

### PR DESCRIPTION
**Pull request by d0c-s4vage**
* `build_events`
* `enable_ssl_verification`

See the two links below:

* https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/projects.md#add-project-hook
* https://github.com/pyapi-gitlab/pyapi-gitlab/pull/173

### CODE REVIEW

**Comment by d0c-s4vage** [comment]
lol build failed b/c:

    git commit title ('Tricky-trickerson python gotcha here - forgotten comma turned two separate strings on different lines into one string.') should not end with period

hrmm

File: gitlab/objects.py [hunk 1]
```diff
@@ -851,7 +851,8 @@ class ProjectHook(GitlabObject):
     requiredUrlAttrs = ['project_id']
     requiredCreateAttrs = ['url']
     optionalCreateAttrs = ['push_events', 'issues_events',
-                           'merge_requests_events', 'tag_push_events']
+                           'merge_requests_events', 'tag_push_events',
+                           'build_events', 'enable_ssl_verification']
```
Comment (gpocentek) [inline gitlab/objects.py]: I don't see build_events in the doc. I see note_events though.

Comment (d0c-s4vage) [inline gitlab/objects.py]: In gitlab-ce code:

https://gitlab.com/gitlab-org/gitlab-ce/blob/master/lib/api/project_hooks.rb#L48-49

**Comment by d0c-s4vage** [comment]
hhhhhhhhh ok, I'll reword that commit again. Never knew there were such stringent restrictions on commit messages.

**Comment by gpocentek** [comment]
I probably should ignore the commit message check, I agree it's a bit annoying.

**Comment by gpocentek** [comment]
Thank you!

**Pull request merged**